guidelines/sc/21/text-spacing.html

```diff
@@ -15,10 +15,8 @@ <h4>Text Spacing</h4>
  
  <p>Exception: Human languages and scripts that do not make use of one or more of these text style properties in written text can conform using only the properties that exist for that combination of language and script.</p>
 
- <p class="note">It is not required that content use the values specified. The requirement is to ensure no loss of content or functionality when a user overrides the authored styles with the values.</p>
+ <p class="note">Content is not required to use these text spacing values. The requirement is to ensure that <strong>when a user overrides</strong> the authored text spacing, content or functionality is not lost.</p>
 
-<p class="note">Some human languages or writing systems have different needs and may adjust other values (such as paragraph start indent) in order to improve legibility. This success criterion might not be sufficient for those needs.</p>
-
-<p class="note">It is beneficial for users if authors use any locally available guidance for improving readability in the local language or writing system. If the user chooses to go beyond the metrics specified, any resulting loss of content or functionality is the user's responsibility.</p>
```

Comment:
The I18N WG feels that something like this is important to include here. I tend to agree that this is maybe not the best way to convey this?

```suggestion
<p class="note">Be aware that there may be additional locally available guidance for improving readability in your specific language or <a>writing system</a> and that users may choose to adjust these values rather than those in this success criterion. While not part of this success criterion, authors should familiarize themselves with local requirements to ensure that such adjustments do not affect functionality.</p>
```

Reply:
Please see edited PR to try to succinctly address key point.